content/departments/product-engineering/engineering/cloud/iam-and-admin-exp/index.md

```diff
@@ -249,17 +249,18 @@ TBD
 
 ### Team slack channels
 
-- [#cloud-saas-internal](https://sourcegraph.slack.com/archives/C02EQBDB1LY) - internal channel for Cloud SaaS team for all day to day communication within the team
-- [#cloud-saas](https://sourcegraph.slack.com/archives/C025BKWFPDY) - external channel for Cloud SaaS team where other Sourcegraphers can ask for help or leave questions for the team
-- [#cloud-saas-jira](https://sourcegraph.slack.com/archives/C02FDFTBATA) - integration with Cloud SaaS Jira project, all changes to the project including new issues, or issue status changes are automatically reported to this channel
+- [#iam-and-admin-exp-internal](https://sourcegraph.slack.com/archives/C02EQBDB1LY) - internal channel for IAM & Admin Exp team for all day to day communication within the team
+- [#iam-and-admin-exp](https://sourcegraph.slack.com/archives/C025BKWFPDY) - external channel for IAM & Admin Exp team where other Sourcegraphers can ask for help or leave questions for the team
+<!-- TODO: slack channel name needs to be updated  first-- will this maybe break  the integration with JIRA?  -->
```

Comment:
Added a TODO there so I won't forget:  need double check if updating the channel's name will break the integration with JIra